src/tests/Microsoft.PowerFx.Interpreter.Tests/ConfigTests.cs

```diff
@@ -680,6 +680,49 @@ public async Task Dataverse()
 
             Assert.Equal(33.0, result.ToObject());
         }
+
+        [Theory]
```

Comment:
Can you add dedicated test for GetMutableCopy() to 
SymbolTableTests.cs

Things to check:
1.  modifying the copy does not change the original.  (ie, it really is a copy, not just a downcast) 
2. calling twice to get 2 copies, mutating them, they don't interfere with each other.  (they really are copies)

3. Things that shouldn't be copied, aren't. Ie, if we only copy functions, then the vars aren't copied.